### PR TITLE
[VTA] add doc to tsim-example driver and update verilator env variable

### PR DESCRIPTION
**Pull request by vegaluisjose**

### CODE REVIEW

File: vta/hardware/chisel/Makefile [hunk 1]
```diff
@@ -15,6 +15,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
+export VERILATOR_INC_DIR = /usr/local/share/verilator/include
```
Comment (tmoreau89) [inline vta/hardware/chisel/Makefile]: This should be external to the Makefile IMO; and set by the user. See: https://docs.tvm.ai/install/from_source.html#tvm-package

I think we should make it clear in the instructions that this variable needs to be set by the user, when they install verilator.

Comment (eqy) [reply vta/hardware/chisel/Makefile]: maybe just wrap the export in an `ifndef`

Comment (vegaluisjose) [reply vta/hardware/chisel/Makefile]: I see. I guess we could try to check if the path exists in the makefile then?.

I think having to setup  more environment variables is a pain for the user.

Comment (tmoreau89) [reply vta/hardware/chisel/Makefile]: I think that hard coded paths will fail at some point depending on os, or package manager. Given that the user will need to install verilator themselves, this is part of the installation process. We just need to add a line saying to export blah.

Comment (vegaluisjose) [reply vta/hardware/chisel/Makefile]: Do you want me to add a comment in the Makefile in the meantime?

Comment (tmoreau89) [reply vta/hardware/chisel/Makefile]: we should have a cascade of checks: check that the variable exists, if not, set to default path; check that the binary exists, else error out

Comment (vegaluisjose) [reply vta/hardware/chisel/Makefile]: Alright, I just added the cascade of checks for binary first then for the path.

Comment (tmoreau89) [reply vta/hardware/chisel/Makefile]: Thanks, it looks good except that we shouldn't use `export` in a Makefile. Instead can we use a `?=` to set if not set?

Comment (vegaluisjose) [reply vta/hardware/chisel/Makefile]: Done

**Pull request merged**